svg.panzoom.js

```diff
@@ -129,16 +143,50 @@ SVG.extend(SVG.Doc, SVG.Nested, {
       lastP = currentP
     }
 
-    this.on('wheel', zoom)
+    this.on('wheel', wheelZoom)
     this.on('touchstart', pinchZoomStart, this, {passive:false})
     this.on('mousedown', panStart, this, {passive:false})
     this.on('touchstart', panStart, this, {passive:false})
 
     return this
 
+  },
+
+  zoom: function(zoomlevel, point) {
+    var isAnimating = this instanceof SVG.FX
+    // get target in case of the fx module, otherwise reference this
+    var target = isAnimating ? this.target() : this
+
+    if(arguments.length === 0) return target.bbox().width / target.viewbox().width
```

Comment:
Albeit svg nodes have a dimension, the bbox is always the closest box around all elements in it. Therefore it is not useful here.